### PR TITLE
[Feature] Debrand and improve spinner

### LINKED ISSUE #182: MFR loading spinner lower resolution than other OSF spinners
I reported this on the 0.46 trello board.

## Steps

1. Log into osf.io.
2. Navigate to one of your projects.
3. Click on a file to render it on the file detail page.
4. Notice the spinner in the file tree and the spinner in the renderer as the page is loading.

## Expected behavior

I expect the spinners to be the same resolution.

## Actual behavior

The spinner in the MFR is lower resolution than other OSF spinners, including the one on the same page in the file tree on the left.

## Screenshots

**File tree spinner**

files_grid_high_resolution_spinner

**MFR spinner**

mfr_low_resolution_spinner

### PR DESCRIPTION
**Pull request by mfraezz**
Purpose
======
* COS/OSF branding is intentionally left out of the user-facing parts of the MFR (error messages, etc.). It follows that it should not be present for the loading icon.
* Add spinning spinner functionality on Mozilla browsers
* Accompanies https://github.com/CenterForOpenScience/osf.io/pull/4146
* Closes #182 

Changes
=======
* Switch default spinner to a non-branded one, branded spinner retained but not default
* Add CSS rules for Mozilla
* Add param to allow for use of non-default spinner images

Side Effects
=========
If an incorrect `img_name` is passed in, the 'broken image' icon will spin. This is likely unavoidable without additional AJAX requests to verify image existence, which would slow it down enough to obviate even *having* a loading icon.

### CODE REVIEW

**Comment by lyndsysimon** [comment]
I wasn't assigned to this, but I had to review it in conjunction with https://github.com/CenterForOpenScience/osf.io/pull/4146. For what it's worth, this looks good to me too.

:star:

File: mfr/server/static/js/mfr.js [hunk 1]
```diff
@@ -46,14 +46,15 @@
         return el;
     }
 
-    function _createSpinner(url) {
+    function _createSpinner(url, img_name) {
```
Comment (icereval) [inline mfr/server/static/js/mfr.js]: `img_name` should be camelCase not snake_case

File: mfr/server/static/js/mfr.js [hunk 2]
```diff
@@ -64,14 +65,15 @@
      * @class Render
      * @param {String} id The id of the div into which the iframe will be rendered.
      * @param {String} url The url of the iframe source.
+     * @param {String} img_name The filename of an image in mfr/server/static/images/ to use as a loading spinner
```
Comment (icereval) [inline mfr/server/static/js/mfr.js]: out of order with function inputs

**Comment by icereval** [comment]
A few minor tweaks and should be good to go.

**Pull request merged**